### ISSUE TITLE
[Windows Powershell/Command Prompt/Azure CloudShell]: The Name property of a focusable element present on the homepage,must not be null.

### ISSUE THREAD

**Issue by Shubham786786** [microsoft/terminal#14385]
### Windows Terminal version

1.17.3052.0

### Windows build number

Windows 11 Version 22H2 (OS Build 25206.1000)

### Steps to reproduce

1. Open Windows Terminal.
2. Run Accessibility insights for Window on the homepage.
3. Observe the issue.

****User Experience**:** -
Screen Reader user are impacted here as name is announced for control as result, they will not be able to know the purpose of control.

**Guideline Reference**:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Attachments:
[The Name property of a focusable element must not be null.zip](https://github.com/microsoft/terminal/files/10002487/The.Name.property.of.a.focusable.element.must.not.be.null.zip)

### Expected Behavior

The Name property of a focusable element must not be null.

### Actual Behavior

The Name property of the control defined in the side of the homepage should not be null and the control should be made hidden.

**Comment by carlos-zamora** [comment]
@Shubham786786 In #14541, I've added a name to this element and I've also made the `AccessibilityView` `Raw`. However, now FastPass is complaining that the `AccessibilityView` should be set to `Control`, but we don't want this element to be interactable or focusable in any way. It's purely an implementation detail. Is there a way to exempt this control from FastPass? Or when you do your next pass, can you manually exempt it and not file a bug on us for it?
